### PR TITLE
Add global testcontainers header to docs

### PR DESCRIPTION
**Pull request by leocross**
## What does this PR do?
Adds a global navigation to the documentation that mirrors the navigation on https://testcontainers.com

## Why is it important?
Testcontainers is a cross-language project and many resources like the getting started article, guides and the modules library that are not specific to a language implementation are hosted one testcontainers.com. Adding a global header with consistent navigation between the project site and the language documentation will help new and existing users discover these resources.

### CODE REVIEW

**Review by eddumelendez — APPROVED** [review]
Looks so cool!

**Review by cristianrgreco — APPROVED** [review]

File: docs/css/extra.css [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+    --md-footer-bg-color: #F7F9FD;
+    --md-footer-bg-color--dark: #F7F9FD;
 }
 
 .card-grid {
     display: grid;
     gap: 10px;
 }
 
-.tc-version {
```
Comment (mdelapenya) [inline docs/css/extra.css:18]: This is class is needed, as all module pages include an HTML span element using it. Please see https://github.com/testcontainers/testcontainers-go/blob/main/docs/modules/k3s.md?plain=1#L3 as an example.

File: docs/css/extra.css [hunk 2]
```diff
@@ -85,7 +84,7 @@ body .card-grid-item:focus {
 }
 
 .community-callout a {
-    transition: opacity 0.2s ease;
+transition: opacity 0.2s ease;
```
Comment (mdelapenya) [inline docs/css/extra.css:87]: I think the indentation was removed accidentally here

File: docs/theme/partials/tc-header.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
+{% set header = ({
+    "siteUrl": "https://testcontainers.com/",
+    "menuItmes": [
```
Comment (mdelapenya) [inline docs/theme/partials/tc-header.html:3]:
```suggestion
    "menuItems": [
```

File: docs/theme/partials/tc-header.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            <svg width="30" height="30" viewBox="0 0 30 30" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                <rect y="22" width="30" height="4" rx="2"/>
+                <rect y="13" width="30" height="4" rx="2"/>
+                <rect y="4" width="30" height="4" rx="2"/>
+            </svg>        
+        </button>
+    </div>
+    <nav>
+        <ul class="menu">
+            {% for menuItem in header["menuItmes"] %}
```
Comment (mdelapenya) [inline docs/theme/partials/tc-header.html:111]:
```suggestion
            {% for menuItem in header["menuItems"] %}
```

**Review by mdelapenya — CHANGES_REQUESTED** [review]
LGTM although I saw a CSS class being removed but still used across all the module pages. I think once that's restored, we can merge.

Thanks!

**Review by mdelapenya — DISMISSED** [review]
LGTM! Thanks!!

@leocross let's talk about improving the CSS for the tc-version block 🙏

**Review by mdelapenya — APPROVED** [review]

**Pull request merged**